File: cpp/program/Human.cpp
#pragma once

#include <string>

using namespace std;

class Human{

protected:
    string nik;
    string name;
    string gender;

public:

    Human(string nik = "", string name = "", string gender = ""){
        this->nik = nik;
        this->name = name;
        this->gender = gender;
    }

    ~Human(){};

    //
    // Setters, to set the values
    //
    void setNik(string nik){
        this->nik = nik;
    }

    void setName(string name){
        this->name = name;
    }

    void setGender(string gender){
        this->gender = gender;
    }

    //
    // Getters, to get val
    //

    string getNik(){
        return this->nik;
    }

    string getName(){
        return this->name;
    }

    string getGender(){
        return this->gender;
    }

};

File: java/program/Human.java


public class Human{

    protected String name;
    protected String nik;
    protected String gender;

    Human(String name, String nik, String gender){
        this.name = name;
        this.nik = nik;
        this.gender = gender;
    }

    public String getName() {
        return name;
    }

    public String getNik() {
        return nik;
    }

    public String getGender() {
        return gender;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setNik(String nik) {
        this.nik = nik;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

};